### DISCUSSION TITLE
redis/Redis%E4%BA%94%E5%A4%A7%E5%9F%BA%E6%9C%AC%E6%95%B0%E6%8D%AE%E7%B1%BB%E5%9E%8B

### DISCUSSION THREAD

**Discussion by giscus** [vuepress-theme-hope/giscus-discussions#889] [Announcements]
# redis/Redis%E4%BA%94%E5%A4%A7%E5%9F%BA%E6%9C%AC%E6%95%B0%E6%8D%AE%E7%B1%BB%E5%9E%8B

一、概述 首先对redis来说，所有的key（键）都是字符串。我们在谈基础数据结构时， 讨论的是存储值的数据类型，主要包括常见的5种数据类型，分别是：String、List、Set、Zset、Hash，还有其他几种特殊数据类型 具体可参考官网文档地址：https://redis.io/docs/data-types/ 二、redis key（键）操作 ...

http://localhost:8080/redis/Redis%E4%BA%94%E5%A4%A7%E5%9F%BA%E6%9C%AC%E6%95%B0%E6%8D%AE%E7%B1%BB%E5%9E%8B.html

**Comment by cxslh** [comment]
123
